README.md

```diff
@@ -0,0 +1,75 @@
+# PG::Instrumentation
+
+This gem provides OpenTracing auto-instrumentation for [ruby-pg](https://github.com/ged/ruby-pg).
```

Comment:
nit, "instrumentations" in keeping in line with the OT project.

Reply:
Please clarify which "instrumentation" should be pluralized. I went through other the README of our other instrumentations and there wasn't couldn't find anything that was pluralized. For example: https://github.com/opentracing-contrib/ruby-mysql2-instrumentation/blob/master/README.md#mysql2instrumentation

Reply:
My request is to remove the "auto-", but singular is fine.